src/ui/components/CarouselSection/index.js

```diff
@@ -0,0 +1,46 @@
+/* @flow */
+import classNames from 'classnames';
+import React from 'react';
+
+import Link from 'amo/components/Link';
+
+import './styles.scss';
+
+
+type PropTypes = {|
+  children?: any,
+  linkTo?: Object | string,
+  styleName: string,
+|};
+
+export default class CarouselSection extends React.Component {
+  static defaultProps = {
+    styleName: 'default',
+  }
```

Comment:
Don't we need a semi-colon here?

Reply:
Nope! Linter would yell at me if I did. 😄

Reply:
ah snap, sorry for the false positive :(